pkgs/swift2objc/lib/src/transformer/transformers/transform_variable.dart

```diff
@@ -73,24 +74,28 @@ Declaration _transformVariable(
       ? originalVariable.hasSetter
       : !originalVariable.isConstant;
 
+  // properties that throw or are async need to be wrapped in a method
   if (originalVariable.throws || originalVariable.async) {
     final prefix = [
       if (originalVariable.throws) 'try',
       if (originalVariable.async) 'await'
     ].join(' ');
 
+    final (type, _) = getWrapperIfNeeded(
```

Comment:
Hmmm. This is a bit suspicious. Should you be calling `maybeWrapValue` here, and using the wrapping code to construct the return statement on line 98?